lib/services/monitor/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
I do not know who owns monitoring, but it looks like a very old version of our API. In any case, I have never touched that code.